### PR TITLE
Add Ruby integration test for monitoring API

### PR DESCRIPTION
**Pull request by swcloud**

### CODE REVIEW

File: test/ruby/monitoring/monitoring.rb [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Integration tests for vkit generated client library for
+# Stackdriver Monitoring APIs.
+#
+require "google/cloud/monitoring/v3"
+
+if ARGV.length != 1
```
Comment (jmuk) [inline test/ruby/monitoring/monitoring.rb]: `ARGV.empty?`

File: test/ruby/monitoring/monitoring.rb [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  display_name: "displayNameUpdate",
+  filter: "resource.type = gce_instance"
+)
+updated_group = group_service_api.update_group(group_to_update)
+puts "Group updated: #{updated_group.display_name}"
+
+#
+# Integration test for GroupServiceApi.get_group
+#
+group_id = created_group.name.split('/')[-1]
```
Comment (jmuk) [inline test/ruby/monitoring/monitoring.rb]: `GroupServiceApi.match_group_from_group_name created_group.name` might be better?

File: test/ruby/monitoring/monitoring.rb (hunk 1 above)
Comment (swcloud) [inline test/ruby/monitoring/monitoring.rb]: Done

File: test/ruby/monitoring/monitoring.rb (hunk 2 above)
Comment (swcloud) [inline test/ruby/monitoring/monitoring.rb]: Done

File: test/ruby/monitoring/monitoring.rb [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+Metric = Google::Api::Metric
+metric = Metric.new(type: "custom.googleapis.com/instance/cpu/usage_time")
+Common = Google::Monitoring::V3
+time_interval = Common::TimeInterval.new(
+  end_time: Google::Protobuf::Timestamp.new(seconds: Time.now.to_i - 1000)
+)
+typed_value = Common::TypedValue.new(bool_value: true)
+point = Common::Point.new(
+  interval: time_interval,
+  value: typed_value
```
Comment (jmuk) [inline test/ruby/monitoring/monitoring.rb]: nit: you can embed like `value: Common::TypedValue.new(bool_value: true)` to reduce a single line.

File: test/ruby/monitoring/monitoring.rb [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+)
+time = Common::TimeSeries.new(
+  metric: metric,
+  points: [point]
+)
+time_series = [time]
+metric_service_api.create_time_series(formatted_project_name, time_series)
```
Comment (jmuk) [inline test/ruby/monitoring/monitoring.rb]: just `metric_service_api.create_time_series(formatted_project_name, [time])`. This is for reducing a single line of code.

File: test/ruby/monitoring/monitoring.rb [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+  formatted_project_name, FILTER, time_interval, view).each do |time_series|
+    puts "Listed time series: #{time_series.resource.type}"
+end
+
+#
+# Integration test for MetricServiceApi.list_monitored_resource_descriptors
+#
+metric_service_api.list_monitored_resource_descriptors(
+  formatted_project_name).each do |monitored_resource_descriptor|
+  puts "Listed monitored_resource_descriptor: #{monitored_resource_descriptor.name}"
```
Comment (jmuk) [inline test/ruby/monitoring/monitoring.rb]: two more spaces at the beginning? (consistent with line 154).

File: test/ruby/monitoring/monitoring.rb [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+#
+metric_service_api.delete_metric_descriptor(formatted_metric_descriptor_name)
+puts "Metric descriptor deleted: #{formatted_metric_descriptor_name}"
+
+#
+# Integration test for MetricServiceApi.create_time_series
+#
+formatted_project_name = MetricServiceApi.project_path(PROJECT_NAME)
+Metric = Google::Api::Metric
+metric = Metric.new(type: "custom.googleapis.com/instance/cpu/usage_time")
```
Comment (jmuk) [inline test/ruby/monitoring/monitoring.rb]: Can we just use the same `METRIC_ID` defined at line 83?

Comment (swcloud) [inline test/ruby/monitoring/monitoring.rb]: done.

File: test/ruby/monitoring/monitoring.rb (hunk 3 above)
Comment (swcloud) [inline test/ruby/monitoring/monitoring.rb]: I'd prefer to have one argument per line.

File: test/ruby/monitoring/monitoring.rb (hunk 4 above)
Comment (swcloud) [inline test/ruby/monitoring/monitoring.rb]: I think using 'time_series' as a separate variable can make the argument a little more clear.

File: test/ruby/monitoring/monitoring.rb (hunk 5 above)
Comment (swcloud) [inline test/ruby/monitoring/monitoring.rb]: done.

**Pull request merged**